installing/disconnected_install/installing-mirroring-creating-registry.adoc

```diff
@@ -21,6 +21,7 @@ If you already have a container image registry, such as Red Hat Quay, you can sk
 * 2 or more vCPUs.
 * 8 GB of RAM.
 * About 12 GB for {product-title} {product-version} release images, or about 358 GB for {product-title} {product-version} release images and {product-title} {product-version} Red Hat Operator images. Up to 1 TB per stream or more is suggested.
+* An OpenSSL certificate for your registry.
```

Comment:
I think the available updates should be installing openssl package like podman, reference to